scripts/whitelist.js

```diff
@@ -0,0 +1,23 @@
+/* eslint no-unused-vars: "off", prefer-const: "off" */
+const {
+  getAccounts, whitelist,
+} = require('./hdwallet');
+
+async function addToWhitelist(address) {
+    const accounts = await getAccounts();
+
+    console.log('Whitelisting address', address);
+    await whitelist(myArgs[0], address);
```

Comment:
This line should be `await whitelist(address, accounts[0]);` address = user input = myArgs[0] so this is passing the same address for both params, and the params for the whitelist function is `(addressToBeWhitelisted, owner)`

Reply:
Good catch, I fixed the parameters in the latest commit.

Reply:
LGTM